modules/OCPP201/OCPP201.cpp

```diff
@@ -866,6 +866,11 @@ void OCPP201::ready() {
                 this->r_extensions_15118.at(extensions_id)->call_set_get_certificate_response(everest_response);
             });
 
+        extension->subscribe_charging_needs([this](const types::iso15118::ChargingNeeds& charging_needs) {
+            this->charge_point->on_ev_charging_needs(
```

Comment:
Please make sure that `std::out_of_range` is caught here for `to_ocpp_notify_ev_charging_needs_request`